langserver/config.go

```diff
@@ -77,6 +96,16 @@ func (c *Config) Validate() error {
 			return err
 		}
 	}
+
+	if len(c.LogFormat) > 0 {
+		if !mapLogFormat[c.LogFormat] {
+			return fmt.Errorf(`invalid value for logFormat. "%s" Valid values are  "text" or "string"`, c.LogFormat)
```

Comment:
valid values are "text" or *"json"* not "string"

To avoid that you could use the string formatting instead like that :

```go
fmt.Errorf("valid values are: %s,%s",TextFormat, JSONFormat)
```

Reply:
Thanks for the hint and the rest of the review!

I just accidentally merged this PR by pushing to the master branch instead of this one, but the push included the change you suggested.

Reply:
lol np :). Well done for the PR !